cli/getting_started.md

```diff
@@ -0,0 +1,210 @@
+# Getting Started
+
+This getting started guide will cover creating a vault, minting some VRT, delegating to an operator, updating the vault and withdrawing VRT.
+
+## Setup
+
+Build and install the CLI
+
+In the root of the repo:
+
+```bash
+cargo build --release
+cargo install --path ./cli --bin jito-restaking-cli
+```
+
+Ensure it has been installed
+
+```bash
+jito-restaking-cli --help
+```
+
+## Create a Vault
+
+This is a Jito Test Vault, which uses JitoSOL as its supported token.
+
+THIS IS FOR TESTING PURPOSES ONLY
+Example Vault: `jkHHVMhQefVuEiFKEyEZgcDZoXv8ZZyjUiK11e61oVY`
+Example VRT: `5rN9m6TkyPkzMGVpdmbRVYct1RKa7VssV1AwsHVPFaxJ`
+Example Operator: `EN7drMzCkZqpuyMVW1QBu8Ciw4Se76KNxNvFZYhDnyUH`
+
+### Initialize Vault
+
+Creating a vault requires:
+
+- `<RPC_URL>`: RPC url
+- `<TOKEN_MINT>`: The pubkey of the "supported token" mint
+- `<DEPOSIT_FEE_BPS>`: Fee for minting VRT
+- `<WITHDRAWAL_FEE_BPS>`: Fee for withdrawing ST
+- `<REWARD_FEE_BPS>`: Fee taken when ST rewards are sent to the vault
+- `<DECIMALS>`: Decimals of the newly created VRT. ( 9 is Recommended )
+
+```bash
+jito-restaking-cli --rpc-url <RPC_URL> vault vault initialize <TOKEN_MINT> <DEPOSIT_FEE_BPS> <WITHDRAWAL_FEE_BPS> <REWARD_FEE_BPS> <DECIMALS>
+```
+
+Note the resulting Vault Pubkey.
+
+### Create VRT Metadata
+
+To create the metadata:
+
+- `<RPC_URL>`: RPC url
+- `<VAULT>`: The vault Pubkey
+- `<NAME>`: Name for VRT
+- `<SYMBOL>`: Symbol for VRT
+- `<URI>`: Metadata url
+
+```bash
+jito-restaking-cli --rpc-url <RPC_URL> vault vault create-token-metadata <VAULT> <NAME> <SYMBOL> <URI>
+```
+
+## Update a Vault
+
+It is the vault's responsibility to update it once per epoch. If a vault is not updated, no other actions can be taken. This is done by initializing a `vault_update_state_tracker`, cranking it and to finish the update, closing it.
+
+### Initialize Vault Update State Tracker
+
+Starts the update process, this should be the first IX called at the start of an epoch.
+
+- `<RPC_URL>`: RPC url
+- `<VAULT>`: The vault Pubkey
+
+```bash
+jito-restaking-cli --rpc-url <RPC_URL> vault vault initialize-vault-update-state-tracker <VAULT>
+```
+
+### Crank Vault Update State Tracker
+
+Needs to be called for each operator. If there are no operators, this IX can be skipped. Operators need to be called in order.
+
+- `<RPC_URL>`: RPC url
+- `<VAULT>`: The vault Pubkey
+- `<OPERATOR>`: The operator that is being updated
+
+```bash
+jito-restaking-cli --rpc-url <RPC_URL> vault vault crank-vault-update-state-tracker <VAULT> <OPERATOR>
+```
+
+### Close Vault Update State Tracker
+
+- `<RPC_URL>`: RPC url
+- `<VAULT>`: The vault Pubkey
+- `[NCN_EPOCH]`: Optional NCN Epoch, for closing older, still-open, vault update state trackers
+
+```bash
+jito-restaking-cli --rpc-url <RPC_URL> vault vault close-vault-update-state-tracker <VAULT> <OPERATOR> [NCN_EPOCH]
+```
+
+## Vault Functions
```

Comment:
no strong feelings either way, but there is 00_cli.md which contains a bunch of stuff. i think makes more sense for this doc to be focused on creating + updating vault. can fill out as needed

Reply:
The point of this doc is to add some more context around the different operations and ordering needed for vault creation and maintenance. For example, it explains the vault update loop as well as all of the steps to create and delegate to an operator. I think it's a helpful resource, that being said, I don't care where it lives. It seems like that `00_cli.md` is generated, so I didn't want to put things there.